src/Tintin/ConfigurationLoading.hs

```diff
@@ -3,54 +3,57 @@ module Tintin.ConfigurationLoading
 where
 
 import Tintin.Core
-import qualified Tintin.Errors as Errors
-import qualified Tintin.Capabilities.Logging as Logging
-import qualified Tintin.Capabilities.Filesystem as Filesystem
+import qualified Tintin.Errors as E (showAndDie)
+import qualified Tintin.Capabilities.Logging as Logging 
+import Tintin.Capabilities.Logging (debug)
+import qualified Tintin.Capabilities.Filesystem as FileSystem
+import qualified Tintin.Capabilities.Filesystem as FS (Path(..), readFile, writeFile, doesExist, currentDirectory, list)
 import qualified Tintin.Domain.HtmlFile as HtmlFile
 import qualified Tintin.Domain.Project as Project
 
-import qualified Data.Text as Text
+import qualified Data.Text as T (head, tail, singleton, toUpper, isPrefixOf, strip, stripPrefix, stripSuffix, takeWhile, isInfixOf)
```

Comment:
In my opinion, **T** is not that idiomatic, is there a reason to modify this? Also, I like the way Haskell has to import explicit functions, but I think in this case they are too much. Leave some comments and we can discuss this! 

PD: Remember the conflicts!